wp-content/themes/core/assets/pcss/typography/_mixins.pcss

```diff
@@ -112,10 +123,10 @@
 
 @define-mixin t-category {
 	color: var(--color-black);
-	font-family: var(--wp--preset--font-family--roboto);
-	font-size: var(--wp--preset--font-size--10);
+	font-family: var(--font-family-sans-serif);
+	font-size: var(--font-size-10);
 	line-height: 1.6;
-	font-weight: var(--wp--custom--font-weight--regular);
+	font-weight: var(--font-weight-regular);
 	text-transform: uppercase;
 	text-decoration: none !important; /* override block editor default */
```

Comment:
Do we still need this override?

Reply:
I believe this is used in the Post Terms block to style category / tag which uses links to display the categories / tags. I believe by default these links use underline, hence the override. Same applies to the override below.